python/src/deltachat/account.py

```diff
@@ -151,6 +159,9 @@ def __init__(self, db_path, logid=None):
         self._evlogger = EventLogger(self.dc_context, logid)
         self._threads = IOThreads(self.dc_context)
 
+    def __del__(self, dc_context_unref=capi.lib.dc_context_unref):
```

Comment:
Is it not better to just not do anything if the module is `None`?  the interpreter is already shutting down anyway... (sure this breaks some sub-interpreter use-cases, but hey)

also i believe this problem magically goes away when using Python 3.x+ (for x being something like 4, 5 or 6) ;)